packages/rum-core/src/domain/rumEventsCollection/resource/resourceCollection.ts

```diff
@@ -68,13 +69,25 @@ function processRequest(
 
   const responseDurationInfo = computeResponseDurationInfo(request)
 
+  const duration = toServerDuration(request.duration)
+  const computedAndEntryDurations = isExperimentalFeatureEnabled('resource_durations')
+    ? {
+        _dd: {
+          computed_duration: duration,
+          performance_entry_duration: correspondingTimingOverrides
+            ? correspondingTimingOverrides.resource.duration
+            : undefined,
+        },
+      }
+    : undefined
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏what about extracting this logic in a function like the other computations?